src/Ui.GraphiQL/Internal/graphiql.cshtml

```diff
@@ -52,8 +52,7 @@
    -->
   <link
     rel="stylesheet"
-    href="https://unpkg.com/graphiql@1.5.1/graphiql.min.css"
-    integrity="sha384-1YHEU+Xy8hlKYAZ26WTz+JQEPMM6i/Mx5m8umMkSZChlzSYmq7RqyCyRbGqrILVZ"
+    href="https://unpkg.com/graphiql/graphiql.min.css"
```

Comment:
I think we should maintain a specific version tag and integrity hash.

Reply:
Sure, updated